autoPyTorch/pipeline/components/setup/augmentation/image/ImageAugmenter.py

```diff
@@ -0,0 +1,135 @@
+import os
+from collections import OrderedDict
+from typing import Any, Dict, Optional, Union
+
+from ConfigSpace.configuration_space import (
+    Configuration,
+    ConfigurationSpace
+)
+
+import imgaug.augmenters as iaa
+
+import numpy as np
+
+from autoPyTorch.pipeline.components.base_component import (
+    ThirdPartyComponents,
+    find_components,
+)
+from autoPyTorch.pipeline.components.setup.augmentation.image.base_image_augmenter import BaseImageAugmenter
+
+
+augmenter_directory = os.path.split(__file__)[0]
+_augmenters = find_components(__package__,
+                              augmenter_directory,
+                              BaseImageAugmenter)
+_addons = ThirdPartyComponents(BaseImageAugmenter)
+
+
+def add_augmenter(augmenter: BaseImageAugmenter) -> None:
+    _addons.add_component(augmenter)
+
+
+def get_components() -> Dict[str, BaseImageAugmenter]:
+    """Returns the available augmenter components
+
+    Args:
+        None
+
+    Returns:
+        Dict[str, BaseImageAugmenter]: all BaseImageAugmenter components available
+            as choices
+    """
+    components = OrderedDict()
+    components.update(_augmenters)
+    components.update(_addons.components)
+    return components
+
+
+class ImageAugmenter(BaseImageAugmenter):
+
+    def __init__(self, random_state: Optional[Union[int, np.random.RandomState]] = None):
+        super().__init__()
+        self.available_augmenters = get_components()  # type: Dict[str, BaseImageAugmenter]
+        self.random_state = random_state
+
+    def fit(self, X: Dict[str, Any], y: Any = None) -> BaseImageAugmenter:
+        self.augmenter = iaa.Sequential([augmenter.fit(X).get_image_augmenter() for _, augmenter in
```

Comment:
Did autopytorch support object detection? else, we will have to consider something like https://imgaug.readthedocs.io/en/latest/source/examples_bounding_boxes.html

Reply:
As far as I am aware, autopytorch didnt suport object detection. I can change it to support object detection if we decide to have that task after refactoring